### PR TITLE
fixes in rex packgage job tests

### PR DESCRIPTION
**Pull request by pondrejk**
Faster setup with custom repo using global registration

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py -k test_positive_install_multiple_packages_with_a_job_by_ip

**Review by ogajduse — APPROVED** [review]
Nice! I would like to see more PRs move our tests toward the `register` method.

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -278,11 +278,14 @@ def test_positive_run_default_job_template_multiple_hosts_by_ip(
         assert result['success'] == '2', output_msgs
 
     @pytest.mark.tier3
+    @pytest.mark.no_containers
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_remoteexecution.py:281]: why we are adding this, I assume this was working for containers ? is that correct?

File: tests/foreman/cli/test_remoteexecution.py [hunk 2]
```diff
@@ -292,30 +295,17 @@ def test_positive_install_multiple_packages_with_a_job_by_ip(self, rex_contentho
...
-        subs = entities.Subscription(organization=self.org).search(
-            query={'search': f'name={prod.name}'}
+        client.register(
+            module_org,
+            None,
+            module_ak_with_cv.name,
+            target_sat,
+            repo=settings.repos.yum_3.url,
+            ignore_subman_errors=True,
+            force=True,
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_remoteexecution.py:307]: As we are registering the host for the first time, we can omit these options.
```suggestion

```

File: tests/foreman/cli/test_remoteexecution.py [hunk 3]
```diff
@@ -756,29 +749,17 @@ def test_positive_run_packages_and_services_job(self, rex_contenthost, module_or
...
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_remoteexecution.py:761]: As we are registering the host for the first time, we can omit these options.


```suggestion

```

**Review by jyejare — APPROVED** [review]

**Review by synkd — APPROVED** [review]

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py -k test_positive_install_multiple_packages_with_a_job_by_ip

**Comment by pondrejk** [comment]
suggestions applied, prt passing, just please do squash merge whoever is merging, thanks!

**Pull request merged**